sequencing/gen_ancestry/dxapp.json

```diff
@@ -51,11 +53,49 @@
     }
```

Comment:
Maybe change the output parameter
"ancestry_out" -> "output_ancestry" to follow the input/output nomenclature?
Also will need to update the variable used on the app script.